study.md

```diff
@@ -111,15 +118,16 @@ The array should contain two strings, "drinking beer" and "propane and propane
 accessories".
 
 ```javascript
-// your answer here
+hank.'middle name' = "Rutherford";
```

Comment:
These will cause an unexpected string error.  If you want to set string keys on objects, then you have to use brackets.  e.g., `hank['middle name'] = ...; hank['hobbies'] = ...;`